public/test/jest-setup.ts

```diff
@@ -20,3 +20,24 @@ configure({ adapter: new Adapter() });
 
 const global = window as any;
 global.$ = global.jQuery = $;
+
+const localStorageMock = (function() {
+  var store = {};
+  return {
+    getItem: function(key) {
+      return store[key];
+    },
+    setItem: function(key, value) {
+      store[key] = value.toString();
+    },
+    clear: function() {
+      store = {};
+    },
+    removeItem: function(key) {
+      delete store[key];
+    },
+  };
+})();
+
+global.localStorage = localStorageMock;
```

Comment:
Local storage would be a good candidate for a redux middleware. Used that with great success at earlier employee.

Reply:
yes! replacing the app/core/store localstorage wrapper we use for some things (like recently visited dashboards) would be great for this!